src/Makefile

```diff
@@ -1,3 +1,5 @@
+OBJS = detections.o
```

Comment:
I would rather call it "detection" rather than "detections". What you think?

Reply:
Ok, changed (to build the project first a make clean should be done to delete old .o)